HopsanCore/src/CoreUtilities/SimulationHandler.cpp

```diff
@@ -102,6 +102,16 @@ bool SimulationHandler::simulateSystem(const double startT, const double stopT,
     return false;
 }
 
+bool SimulationHandler::startRealtimeSimulation(const double startT, ComponentSystem *pSystem, double realtimeFactor)
```

Comment:
startT not used ?, also no need to make it const, functions taking value by copy should be allowed to do what they want with it

Reply:
Ok, I will remove it. The `startT` argument in `simulateSystem` is needed only for multi-threaded simulations, because they need to reinitialize the model after measuring simulation time. If we want parallel realtime simulation in the future we could need to add it to `startRealtimeSimulation` again.